uberfire-extensions/uberfire-layout-editor/uberfire-layout-editor-client/src/main/java/org/uberfire/ext/layout/editor/client/components/rows/Row.java

```diff
@@ -855,6 +866,61 @@ private void setupColumnResizeActions() {
         }
     }
 
+    private void setupCssProperties() {
+        List<CssValue> cssValueList = layoutCssHelper.readCssValues(properties);
+        view.applyCssValues(cssValueList);
+    }
+
+    @Override
+    public Map<String, String> getProperties() {
+        return properties;
+    }
+
+    @Override
+    public void setProperty(String property, String value) {
+        properties.put(property, value);
+        setupCssProperties();
+    }
+
+    @Override
+    public void removeProperty(String property) {
+        properties.remove(property);
+        setupCssProperties();
+    }
+
+    @Override
+    public void clearProperties() {
+        properties.clear();
+        setupCssProperties();
+    }
+
+    @Override
+    public List<PropertyEditorCategory> getPropertyCategories() {
+        List<PropertyEditorCategory> result = new ArrayList<>();
+
+        PropertyEditorCategory category = layoutCssHelper.createCategory(LayoutEditorCssHelper.CSS_CATEGORY_PANEL);
```

Comment:
Could you please extract this and the container getPropertyCategories() block to a helper class?